BoardHandler/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/service/BoardHandler.java

```diff
@@ -32,8 +35,8 @@ void markMiss(Player player, Coordinate coordinate) {
     }
 
     FieldState getOpponentFieldStatus(Player player, Coordinate coordinate) throws NoSuchPlayerException, NoMastAtPositionException {
```

Comment:
check why NoMastAtPositionException is here?

Reply:
true story, fixed